### PR TITLE
Add information about tracking PyPI's upcoming changes

### LINKED ISSUE #3139: Add "how do I keep up?" to FAQ
The [help page/FAQ](https://pypi.org/help/) should include something like:

> Q. How do I keep up with upcoming changes to PyPI?

> A. Subscribe to [the pypi-announce mailing list](https://mail.python.org/mm3/mailman3/lists/pypi-announce.python.org/) and [read PyPI-related blog posts on the PSF blog](https://pyfound.blogspot.com/search/label/pypi) ([Atom feed](https://pyfound.blogspot.com/feeds/posts/default/-/pypi), [RSS feed](https://pyfound.blogspot.com/feeds/posts/default/-/pypi?alt=rss)).

(At some future point, this will expand to include, for instance, https://www.pypa.io/en/latest/roadmap/ . But for right now, the announcement mailing list and the PSF blog posts labelled "pypi" are enough.)

--------

**Good First Issue**: This issue is good for first time contributors. If you've already contributed to Warehouse, please work on [another issue without this label](https://github.com/pypa/warehouse/issues?utf8=%E2%9C%93&q=is%3Aissue+is%3Aopen+-label%3A%22good+first+issue%22) instead. If there is not a corresponding pull request for this issue, it is up for grabs. For directions for getting set up, see our [Getting Started Guide](https://warehouse.pypa.io/development/getting-started/). If you are working on this issue and have questions, please feel free to ask them here, [`#pypa-dev` on Freenode](https://webchat.freenode.net/?channels=%23pypa-dev), or the [pypa-dev mailing list](https://groups.google.com/forum/#!forum/pypa-dev).

### PR DESCRIPTION
**Pull request by pgadige**
Fixes #3139 
Provide links for the reader to keep track of PyPI's upcoming changes.

### CODE REVIEW

File: warehouse/templates/pages/help.html [hunk 1]
```diff
@@ -328,6 +330,11 @@ <h3 id="contributing">{{ contributing() }}</h3>
           <em>Stay updated</em>: You can also follow the ongoing development of the project on the <a href="https://mail.python.org/mailman/listinfo/distutils-sig">Distutils-sig mailing list</a> and the <a href="https://groups.google.com/forum/#!forum/pypa-dev">PyPA Dev message group</a>.
         </p>
         {{ code_of_conduct() }}
+
+        <h3 id="upcoming-changes">{{ upcoming_changes() }}</h3>
+        <p>
+          Subscribe to <a href="https://mail.python.org/mm3/mailman3/lists/pypi-announce.python.org/">the pypi-announce mailing list</a> and <a href="https://pyfound.blogspot.com/search/label/pypi">read PyPI-related blog posts on the PSF blog</a> (<a href="https://pyfound.blogspot.com/feeds/posts/default/-/pypi">Atom feed</a>, <a href="https://pyfound.blogspot.com/feeds/posts/default/-/pypi?alt=rss">RSS feed</a>).
```
Comment (jonparrott) [inline warehouse/templates/pages/help.html]: I recommend we word this a bit differently:

```html
Changes to PyPI are generally announced on both the the <a href="https://mail.python.org/mm3/mailman3/lists/pypi-announce.python.org/">pypi-announce mailing list</a> and the <a href="https://pyfound.blogspot.com/search/label/pypi">PSF blog</a> under the label "pypi". The PSF blog also has <a href="https://pyfound.blogspot.com/feeds/posts/default/-/pypi">Atom</a> and <a href="https://pyfound.blogspot.com/feeds/posts/default/-/pypi?alt=rss">RSS</a> feeds for the "pypi" label.
```

Also please make sure all external links have `rel="noopener"` and `target="_blank"`.

**Comment by brainwane** [comment]
@pgadige I agree with @jonparrott :)

File: warehouse/templates/pages/help.html (hunk 1 above)
Comment (brainwane) [reply warehouse/templates/pages/help.html]: Agreed.

**Comment by nlhkabu** [comment]
This looks ready to re-review @brainwane

**Comment by pgadige** [comment]
Thank you for the review and useful feedback @jonparrott I'm glad I got to learn the importance of `rel="noopener"` and `target="_blank"`

I also got to explore `git rebase` command through working on desired changes. I could successfully rebase my feature branch onto the updated master branch of my local repo.

Thank you @brainwane and @nlhkabu for patiently answering my questions.

**Comment by nlhkabu** [comment]
Thanks all! Merged :tada:

**Comment by brainwane** [comment]
@pgadige Thanks for your persistence and for letting us know what you've learned! 

Next, would you like to help test a pull request? If so, take a look at https://warehouse.readthedocs.io/development/reviewing-patches/#testing-branches-on-your-local-machine and https://warehouse.readthedocs.io/development/getting-started/#running-tests-and-linters , then locally pull and test the branches for #3487 or #3485 and comment there to say whether the branch works for you (check whether the branch functions as its author intended, and whether you see any new unintended bugs when you try clicking around and using stuff).

**Comment by pgadige** [comment]
I'd love to! I shall test the given pull requests and update with my findings. Thank you @brainwane

**Pull request merged**